media/utils/addBlametoDoc.js

```diff
@@ -22,9 +22,9 @@ const addBlametoDoc = (blameArray, commitLink) => {
     </table>
     `);
   //sort array by date, newest first
-  blameArray.sort((a, b) => {
-    return new Date(b.commitDate) - new Date(a.commitDate);
-  });
+  // blameArray.sort((a, b) => {
```

Comment:
Why don't we sort it by newest first now?

Reply:
Moved the sort to main.js. This is so that we can send a single blame line, and be sure that it's the one that's most recent. WDYT about the approach?

Reply:
Would you please point me to the line that sorts on `main.js`

Reply:
Ah, I see it, but that doesn't kill encapsulation?